open-ce/env_config.py

```diff
@@ -17,18 +18,46 @@
           " for a list of requirements.")
     sys.exit(1)
 
+class Key: #pylint: disable=no-init,too-few-public-methods
+    '''Enum for Env Config Keys'''
+    IMPORTED_ENVS = 0
+    CHANNELS=1
+    PACKAGES=2
+    GIT_TAG_FOR_ENV=3
+    GIT_TAG=4
+    FEEDSTOCK=5
+
+ENV_CONFIG_KEYS = {
+    Key.IMPORTED_ENVS: "imported_envs",
+    Key.CHANNELS: "channels",
+    Key.PACKAGES: "packages",
+    Key.GIT_TAG_FOR_ENV: "git_tag_for_env",
+    Key.GIT_TAG: "git_tag",
+    Key.FEEDSTOCK: "feedstock"
+}
+
+_PACKAGE_SCHEMA ={
+    ENV_CONFIG_KEYS[Key.FEEDSTOCK]: utils.make_schema_type(str, True)
+}
+
+_ENV_CONFIG_SCHEMA = {
+    ENV_CONFIG_KEYS[Key.IMPORTED_ENVS]: utils.make_schema_type([str]),
+    ENV_CONFIG_KEYS[Key.CHANNELS]: utils.make_schema_type([str]),
+    ENV_CONFIG_KEYS[Key.GIT_TAG_FOR_ENV]: utils.make_schema_type(str),
+    ENV_CONFIG_KEYS[Key.GIT_TAG]: utils.make_schema_type(str),
```

Comment:
This should actually go under _PACKAGE_SCHEMA

Reply:
Just to be clear: `GIT_TAG` should go under _PACKAGE_SCHEMA.

Reply:
Done.